app/v1/connectors/BFLossConnector.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.connectors
+
+import config.AppConfig
+import javax.inject.{Inject, Singleton}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+import v1.connectors.httpparsers.StandardDesHttpParser._
+import v1.models.des.{AmendBFLossResponse, CreateBFLossResponse, ListBFLossesResponse, RetrieveBFLossResponse}
+import v1.models.domain.{AmendBFLoss, BFLoss}
+import v1.models.requestData._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class BFLossConnector @Inject()(http: HttpClient, appConfig: AppConfig) extends DesConnector {
```

Comment:
(Minor) Either BFLossesConnector to match LossClaimsConnector, or rename LossClaimsConnector to LossClaimConnector for consistency.

Reply:
why? they both are completely different... I created based on the URL in tech spec